src/deploy.py

```diff
@@ -24,13 +27,19 @@ def migrate_schema(db_password):
 
 def generate_passwords() -> Dict[str, str]:
     return {
-        "api": "",
+        "api": ''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(50)),
         # "schema_migrator": ""
     }
 
 
 def set_passwords_for_db_users(passwords):
-    pass
+    conn_string = "host='localhost' port='5432' dbname='montagu' user='vimc' password='changeme'"
+    with psycopg2.connect(conn_string) as conn:
+        with conn.cursor() as cur:
+            cur.execute("ALTER USER vimc WITH PASSWORD '{}'".format(passwords["api"]))
+            conn.commit()
+    # unlike file objects or other resources, exiting the connection’s with block doesn’t close the connection
+    conn.close()
```

Comment:
So should we also have a try finally here? Is it bad if you don't close the connection, or is that just fine?